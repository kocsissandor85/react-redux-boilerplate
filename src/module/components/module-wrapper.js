import React from "react";
import styled from "styled-components";

const StyledDiv = styled.div`
  padding: 1rem;
`;

export const Wrapper = props => (
  <StyledDiv>
    <h1>Yet Another React-Redux Boilerplate</h1>
    {props.children}
  </StyledDiv>
);