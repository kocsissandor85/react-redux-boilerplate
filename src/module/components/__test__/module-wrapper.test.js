import { Wrapper } from "../module-wrapper";

describe("The module wrapper", () => {

  it("renders correctly", () => {
    const wrapper = mount(
      <Wrapper>
        <p>Hello World!</p>
      </Wrapper>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });

});