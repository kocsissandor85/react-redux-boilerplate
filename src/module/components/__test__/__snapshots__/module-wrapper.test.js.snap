// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The module wrapper renders correctly 1`] = `
.c0 {
  padding: 1rem;
}

<Wrapper>
  <module-wrapper__StyledDiv>
    <div
      className="c0"
    >
      <h1>
        Yet Another React-Redux Boilerplate
      </h1>
      <p>
        Hello World!
      </p>
    </div>
  </module-wrapper__StyledDiv>
</Wrapper>
`;
